Automattic/jetpack
# AI Assistant: Fix flickering when adding word to Breve dictionary

Fixes #39086 

## Proposed changes:

* Changes `reloadDictionary` to use a dedicated boolean flag

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to a post with many blocks, having the same typo on multiple blocks
* Add this typo to the dictionary
* Check that it happens immediately
* Check that all instances of the word are de-highlighted (is that even a word? if not, it is now, language is built by its users)
